dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -165,7 +163,8 @@ GetFileTypeIconsByKey(HKEY hKey, PFILE_TYPE_ENTRY Entry)
         return FALSE;
 
     // Get the icon location
-    WCHAR szLocation[MAX_PATH + 32] = { 0 };
+    WCHAR szLocation[MAX_PATH + 32];
+    szLocation[0] = 0;
```

Comment:
Why did you change this?

Reply:
I'm not sure it will really make a difference.

Reply:
Revert or not [R/n]?

Reply:
Please revert.

Reply:
OK.